packages/roosterjs-content-model-plugins/lib/edit/EditPlugin.ts

```diff
@@ -181,7 +181,8 @@ export class EditPlugin implements EditorPlugin {
                     break;
 
                 case 'Enter':
-                    if (!hasCtrlOrMetaKey) {
+                    if (!hasCtrlOrMetaKey && event.rawEvent.keyCode == 13) {
```

Comment:
Interesting. If we are sure that keyCode will be 229 for IME enter, I prefer to check keyCode != 229 here which makes it more clear what it is doing.

Please add a const declaration for this key name and add comment

Reply:
Added